### DISCUSSION TITLE
Drawdown reduction

### DISCUSSION THREAD

**Discussion by nikneural** [dcajasn/Riskfolio-Lib#110] [Q&A]
Hi, I wanted to know what is the best metric to use to reduce drawdown. If I count the portfolio for some period, they give a small drawdown, then the next month they give x3 drawdown, if there is a way, how to avoid this?

**Comment by dcajasn** [comment]
Hi @nikneural,

It depends on what do you want, each drawdown risk measure has advantages and disadvantages based on their definitions, but to reduce drawdowns you have several options:  

- Removing high volatility assets from your asset universe.
- Adding constraints on max weights to reduce concentration.
- Adding constraints on other risk measures.
- etc.

Best,
Dany
